### ISSUE TITLE
Should you use colons at the end of a form field label?

### ISSUE THREAD

**Issue by NancySemple** [MicrosoftDocs/microsoft-style-guide#205]
Hi, I have been searching everywhere for accessibility recommendations about whether to include a colon at the end of a label for a form field on a dialog box. In the past, our products have always included the colon for the labels that appear to the left or above an input box (ie “First name:”). For our newer products, people are now wondering if we should be removing the colon for a more “modern” look. I am unsure what impact this will have on accessibility and find that there is no consistency out there in what commonly used products are doing for their labels (sometimes they seem to include colons, other times they do not). I am also not sure whether there is a specific aria label tag that could be associated to form controls so that the colons would not be necessary from an accessibility standpoint and screen readers. Could you please provide some guidance on whether it is best to include colons at the end of the label for a form field from an accessibility standpoint or if there is some other way in the underlying code that would handle any issues that a screen reader may have not understanding that a label is associated to the field? I do not see the use of colons mentioned anywhere in your guidelines, but it may be that I just haven’t been able to find it. (You do have some information about labels and labeled by, but I’m not sure that addresses the issue?) Thank you.
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a6ab9a7e-9f2f-830d-d941-e8bf363513f6
* Version Independent ID: ebbe8938-b057-34b7-9e53-8e0d90675508
* Content: [Accessibility guidelines and requirements - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/accessibility/accessibility-guidelines-requirements)
* Content Source: [styleguide/accessibility/accessibility-guidelines-requirements.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/accessibility/accessibility-guidelines-requirements.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**
